images/router/haproxy/conf/haproxy-config.template

```diff
@@ -32,6 +35,10 @@ global
 defaults
   # maxconn 4096
   # Add x-forwarded-for header.
+{{ if ne (env "ROUTER_SYSLOG_ADDRESS" "") ""}}
+  option httplog
```

Comment:
I am not sure if it would be beneficial. My understanding is that HAProxy is used as an HTTP proxy inside of openshift and according to the HAProxy documentation httplog is recommended for HTTP proxies